### DISCUSSION TITLE
Ability to set log level in .percy.json

### DISCUSSION THREAD

**Discussion by tmcconechy** [percy/cli#612] [Ideas]
## The problem

We generally only run percy on ci and when we run tests locally we get a lot of log errors. like..

```
 PASS  test/ids-list-builder/ids-list-builder-percy-test.js
[percy] Percy is not running, disabling snapshots
 PASS  test/ids-hyperlink/ids-hyperlink-percy-test.js
[percy] Percy is not running, disabling snapshots
 PASS  test/ids-summary-field/ids-summary-field-percy-test.js
[percy] Percy is not running, disabling snapshots
```

The solution we have it to set the PERCY_LOGLEVEL on the command line.

`PERCY_LOGLEVEL=silent npx jest --coverage`

The problem is this isnt cross platform so we need `set PERCY_LOGLEVEL=silent` on windows.

I tried to set this on .percy.json but either i dont know how or it doesnt work. Is there already a way to set this by config file? If not would it be worth adding?

## Environment

- Node version: 16
- `@percy/cli` version: 1.0.0-beta.70
- Version of Percy SDK you’re using:
- If needed, a build or snapshot ID:
- OS version: windows 11/ mac big sur
- Type of shell command-line [interface]: zshrc / windows

## Code to reproduce issue

- run `PERCY_LOGLEVEL=silent npx jest --coverage` on windows command line

**Comment by Robdel12** [comment]
Hey @tmcconechy! You can use something like `cross-env` to set env vars for many env types: https://www.npmjs.com/package/cross-env Or you could use a `.env` file to set `PERCY_LOGLEVEL`, too. 

What SDK are you using? It should ideally only log once and disable itself. Could be the way the test suite is bundled -- if each test is its own bundle then the SDK wouldn't be able to cache & disable itself.

**Reply by tmcconechy** [reply]
Yeah i did think of cross-env.. Was kind of turned off because its "readonly" tho. I will try the .env file ...

Im using jest and percy cli latest and it seems to log it for every single test (500).

**Reply by Robdel12** [reply]
> and it seems to log it for every single test (500).

Hm, that's not right. Something might be up with how the tests are being bundled together. Which SDK? `@percy/puppeteer`? In any case, all JS SDKs use the same methods. We cache the result of the health check and shouldn't check again: https://github.com/percy/cli/blob/master/packages/sdk-utils/src/percy-enabled.js#L31

This can go "wrong" if each test is it's own bundle and gets its own SDK included many times. That would make it so each test does its own health check and log each time.

**Reply by tmcconechy** [reply]
Oh sorry yes its `@percy/puppeteer` this is the repo and test command if that means anything to you?  https://github.com/infor-design/enterprise-wc/blob/main/package.json#L29
